anchor/anchorRecord.go

```diff
@@ -52,19 +60,23 @@ type EthereumStruct struct {
 var _ interfaces.Printable = (*AnchorRecord)(nil)
 var _ interfaces.IAnchorRecord = (*AnchorRecord)(nil)
 
-func (e *AnchorRecord) JSONByte() ([]byte, error) {
-	return primitives.EncodeJSON(e)
+// JSONByte returns a []byte of the AnchorRecord encoded in Json: nil, error returned upon error
```

Comment:
We should remove the `: nil, error returned upon error`

It's obvious in the return fields.

Reply:
done